src/mixins/text_style.mixin.js

```diff
@@ -63,31 +63,42 @@
       if (!this.styles || !property || property === '') {
         return false;
       }
-      var obj = this.styles, stylesCount = 0, letterCount, foundStyle = false, style,
-          canBeSwapped = true, graphemeCount = 0;
+      var obj = this.styles, stylesCount = 0, letterCount, style,
+          allStyleObjectPropertiesMatch = true, graphemeCount = 0, styleObject;
```

Comment:
what do you think about:
style  => stylePropertyValue

Reply:
clearly I love verbose names!